website/project/model.py

```diff
@@ -3326,7 +3326,7 @@ def add_contributors(self, contributors, auth=None, log=True, save=False):
         if save:
             self.save()
 
-    def add_unregistered_contributor(self, fullname, email, auth, send_email='default', visible=True, permissions=None, save=False):
+    def add_unregistered_contributor(self, fullname, email, auth, send_email='default', visible=True, permissions=None, save=False, user_created=None):
```

Comment:
Let's call this `existing_user` rather than `user_created`, since you're passing in a user object or None, right?